app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -303,29 +316,29 @@ class BookmarksToolbar extends React.Component {
             bookmarkKey={bookmarkKey}
           />)
       }
-      <div
+      <button
         className={css(
           styles.bookmarksToolbar__overflowIndicator
         )}
+        data-bookmarks-overflow-indicator
+        onClick={this.onMoreBookmarksMenu}
       >
-        <BrowserButton
-          bookmarksOverflowIndicator
-          iconOnly
-          size={globalStyles.spacing.bookmarksToolbarOverflowButtonWidth}
-          iconClass={globalStyles.appIcons.angleDoubleRight}
-          onClick={this.onMoreBookmarksMenu}
+        <svg className={css(styles.bookmarksToolbar__overflowIndicator__icon)} xmlns='http://www.w3.org/2000/svg'viewBox='0 0 320 512'>
```

Comment:
is there any benefit of inlining SVGs instead of requiring the file here? if perf is the same I'd prefer the latter for readability.

Reply:
Benefit is:
- no extra network request
- ability to style through css

Sure, it can be imported as a separate component, e.g.  `<BookmarksToolbarOverflowIndicatorIcon />` and since this icon is changing in the icon project and I'm using this technique for most new icons, then I'll be sure to favor not having the `<svg>` paths make the other components look messy. I'd prefer to make that change for that release if that's alright with everyone else.

Reply:
being able to style through CSS is a huge win- I've fought angrily with SVGs when used as a value for `background-image` (and similar use-cases)

Reply:
ya sorry missed the className prop here